src/util/address.rs

```diff
@@ -581,47 +658,26 @@ impl Address {
 // be used in QR codes, see [Address::to_qr_uri]
 impl fmt::Display for Address {
```

Comment:
In my previous push I somehow dropped this part where I reuse the AddressEncoding formatting instead of copy/pasting. Added back now, PTAL.